packages/syntax-core/src/LinkButton/LinkButton.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import { type StoryObj, type Meta } from "@storybook/react";
+import LinkButton from "./LinkButton";
+import FavoriteBorder from "@mui/icons-material/FavoriteBorder";
+
+export default {
+  title: "Components/LinkButton",
+  component: LinkButton,
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/p7LKna9JMU0JEkcKamzs53/Cambly-Design-System-(Draft)?node-id=994%3A2346",
+    },
+  },
+  argTypes: {
+    color: {
+      text: {
+        control: { type: "text" },
+      },
+      href: {
+        control: { type: "text" },
+      },
+      target: {
+        control: { type: "text" },
+      },
+      options: [
+        "primary",
+        "secondary",
+        "tertiary",
+        "destructive-primary",
+        "destructive-secondary",
+        "destructive-tertiary",
+        "success",
+        "branded",
+      ],
+      control: { type: "radio" },
+    },
+    size: {
+      options: ["sm", "md", "lg"],
+      control: { type: "radio" },
+    },
+    fullWidth: {
+      control: "boolean",
+    },
+    onClick: { action: "clicked" },
+  },
+  tags: ["autodocs"],
+} as Meta<typeof LinkButton>;
+
+export const Default: StoryObj<typeof LinkButton> = {
+  args: {
+    text: "Call to action",
+    href: "www.google.com",
```

Comment:
@somethiiing Seems like this links to `https://syntax-git-linkbutton-add-component-cambly.vercel.app/www.google.com`. Could we change the `www.google.com` to `https://www.cambly.com`?